Source/WebCore/platform/graphics/ca/cocoa/PlatformCALayerCocoa.mm

```diff
@@ -313,6 +313,16 @@ - (void)setOwner:(WebCore::PlatformCALayer*)owner
         m_customSublayers = makeUnique<PlatformCALayerList>(tileController->containerLayers());
     }
 
+#if HAVE(CALAYER_ALLOWS_SORTS_SUBLAYERS)
```

Comment:
Can I skip these checks, since the respondsToSelector check should be sufficient?

Reply:
I think it's best to keep them; makes it clearer what code can be removed in future.